pkg/policy/repository_test.go

```diff
@@ -1844,7 +1844,7 @@ Ingress verdict: allowed
 	ctx = buildSearchCtx("bar", "foo", 0)
 	expectedOut = `
 Resolving ingress policy for [any:foo]
-0/1 rules selected
+0/0 rules selected
```

Comment:
There aren't 0 rules in the repository, though?

I think we just need to count the total early on, and print that here. the `0/1` is correct (no rules select 'foo').

Reply:
So the latter number should be the total number of rules in the repository? And the first number the rules selected for "foo"?

Reply:
Fixed :-)